src/editor/editor.cpp

```diff
@@ -213,62 +214,44 @@ namespace Editor
             }
             ImGui::EndPopup();
         }
-        else if (ImGui::BeginPopupModal("TilesetLoadSuccess"))
-        {
-            ImGui::Text("Image loaded with height %i,\n width %i, channels %i!", tileset_height, tileset_width,
-                        tileset_channels);
-            ImGui::SetCursorPos(ImVec2{80, 50});
-            if (ImGui::Button("Ok", ImVec2(120, 0)))
-            {
-                ImGui::CloseCurrentPopup();
-            }
-            ImGui::EndPopup();
-        }
     }
 
     //--------------------------------------------------------
-    void
-    LoadTilesetPopupWindow()
+
+    void LoadTileSet()
```

Comment:
I'd probably just put the contents of this function in the place where this function is being called since its so simple and is confusing in naming (This function is called LoadTileSet() which calls another function TilesetLoad())